source/Interfaces/MainInterface.cpp

```diff
@@ -115,13 +144,47 @@ namespace i_2D {
                                       (iterY < grid.GetHeight() - 1 && symbol_grid[iterY + 1][iterX] == '#');
                 SwitchCellSelect(cellRect, cell, symbol, isVerticalWall);
 
+
             }
+
         }
+
         // Display everything
         mMenu.drawto(mWindow);
+        mTextBox->DrawTo(mWindow);
+        mMessageBoard->drawTo(mWindow);
         mWindow.display();
     }
     /**
+ * @brief Creates a 9x23 window of the symbol grid centered around the player's position.
```

Comment:
I'm missing where 9 and 23 are coming from. Are they related to `COL` and `ROW`? If so, we might want to make those names a little more clear (e.g., `NUM_ROWS` or something)

Reply:
Ah, found them in the header. I do think the name could be a little more clear.